Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM python:3.11
+FROM python:alpine3.17
```

Comment:
I suggest sticking with `python:3.11` or `python:3.11-slim` for a size optimized image. Some science libraries don't work on alpine unfortunately. It has a different c compiler.

Reply:
```suggestion
FROM python:alpine3.11
```